src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -184,7 +184,7 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     # iot hub device stream commands
     with self.command_group('iot hub devicestream', client_factory=iot_hub_service_factory,
-                            min_api="2019-07-01-preview") as g:
+                            min_api="2019-07-01-preview", is_preview=True) as g:
```

Comment:
It is not recommended to downgrade GA commands to preview state. I'd like to confirm with you that the reason for changing it to preview is because the REST service contains new instability factors after using the new preview api-version, right?

Reply:
The api has never been been in GA state as far as I recall. This is also an ask https://github.com/Azure/azure-cli/issues/24554

Reply:
OK, so this command was incorrectly declared GA states before, which was not in line with the expectation, right?

Reply:
Yea, the mistake was not classifying it as preview when it was released originally.

Reply:
OK, got it.